canvas_modules/common-canvas/src/toolbar/toolbar.jsx

```diff
@@ -276,13 +276,11 @@ class Toolbar extends React.Component {
 		this.leftBar = this.props.config.leftBar || [];
 		this.rightBar = this.props.config.rightBar || [];
 		this.rightBar = [...this.rightBar].reverse() || [];
-
 		const leftItems = this.generateToolbarItems(this.leftBar, false, true);
 		const rightItems = this.generateToolbarItems(this.rightBar, false, false);
-
```

Comment:
Adjusted.

Reply:
<img width="855" alt="image" src="https://user-images.githubusercontent.com/89609176/181653097-54f83025-8597-408a-91d1-9316e7ebbdc4.png">

still persisting even though i put it back, not sure why..I'm working it

Reply:
Sounds like it might be your editor that is removing the lines.  Check to see if there is an editor setting that does that kind of thing. You need to be working with an environment that doesn't automatically change the code.